src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+
+![Navigate to apps](../../images/build-an-app/nav-to-apps.png)
+
+</Step>
+
+<Step>
+
+Click the **Victory Charts** Nerdpack in the catalog:
+
+![Victory Charts Nerdpack](../../images/build-an-app/victory-charts-nerdpack.png)
+
+</Step>
+
+<Step>
+
+Go to the Nerdpack's source code repository:
+
+![Victory Charts repo](../../images/build-an-app/victory-charts-repo.png)
+
+All open source Nerdpacks in the catalog have links to their source code in their catalog information.
+
+</Step>
+
+<Step>
+
+Clone the repository:
+
+```sh
+nr1 nerdpack:clone -r https://github.com/newrelic/nr1-victory-visualizations.git
+```
+
+Now you have a local version of the **Victory Charts** Nerdpack! Notice that you used `nr1 nerdpack:clone` instead of `git clone` to copy the repo. `nr1 nerdpack:clone` offers built in functionality to help keep your local copy distinct from the original Nerdpack in the New Relic One public catalog. Specifically, it generates a new Nerdpack UUID so you don't have to do this yourself:
+
+```sh
+[output] Re-generating UUID...
+[output] Committing new UUID...
+```
+
+If you change to the _nr1-victory-visualizations_ directory, and look at the git log, you'll see the new commit:
+
+```sh
+git log -1 -p
+[output] {blue}commit e356bb5b10c3ecc8f93bae66d5739e1676ee21ef {plain}({green}HEAD -> main{plain})
+[output] Author: New Relic CLI <nr1@newrelic.com>
+[output] Date:   Tue May 25 14:29:37 2021 -0400
+[output]
+[output]     "chore: Auto-generated UUID"
+[output]
+[output] diff --git a/nr1.json b/nr1.json
+[output] index 054de52..7a107b5 100644
+[output] --- a/nr1.json
+[output] +++ b/nr1.json
+[output] @@ -1,6 +1,6 @@
+[output]  {
+[output]    "schemaType": "NERDPACK",
+[output] {red}-  "id": "cf5f13d9-815f-4907-a21d-83d02fa2a4fb",
+[output] {green}+  "id": "ab123c45-678d-9012-efg3-45hi6jkl7890",
+[output]    "displayName": "Victory charts visualizations",
+[output]    "description": "Visualizations built on top of Victory charts"
+[output]  }
+```
+
+</Step>
+
+</Steps>
+
+## Customize a Nerdpack
+
+Tweak the **Circular progress bar** visualization to use straight edges and customizable colors.
+
+**Circular progress bar** [renders a `VictoryPie`](https://github.com/newrelic/nr1-victory-visualizations/blob/267251d936dc607a8a9eba5a19ccbd889f092fdb/visualizations/circular-progress-bar/index.js#L158) with some predefined fields. The fields you'll tweak are:
+
+- [colorScale](https://formidable.com/open-source/victory/docs/victory-pie/#colorscale)
+- [cornerRadius](https://formidable.com/open-source/victory/docs/victory-pie/#cornerradius)
+
+<Steps>
+
+<Step>
+
+In your local Nerdpack, open _nr1-victory-visualizations/visualizations/circular-progress-bar/nr1.json_.
+
+_nr1.json_ is the **Circular progress bar** visualization's [metadata file](http://localhost:8000/explore-docs/custom-viz/configuration-options/). Use this file to add a configurable `colorScale` option, which corresponds to the `colorScale` field on `VictoryPie`.
```

Comment:
Looks like this link address is a local host. Think we're looking for https://developer.newrelic.com/explore-docs/custom-viz/configuration-options/, yeah?